utopia-vscode-common/src/mailbox.ts

```diff
@@ -70,27 +70,44 @@ async function initOutbox(outboxToUse: Mailbox): Promise<void> {
   }
 }
 
-async function receiveMessage<T>(messageName: string, parseMessage: (msg: string) => T): Promise<T> {
+async function receiveMessage<T>(
+  messageName: string,
+  parseMessage: (msg: string) => T,
+): Promise<T> {
   const messagePath = pathForInboxMessage(messageName)
   const content = await readFileSavedContentAsUTF8(messagePath)
   return parseMessage(content)
 }
 
+async function waitForPathToExist(path: string, maxWaitTime: number = 5000): Promise<void> {
+  if (maxWaitTime >= 0) {
+    const doesItExist: boolean = await exists(path)
+    if (!doesItExist) {
+      await waitForPathToExist(path, maxWaitTime - 100)
```

Comment:
```suggestion
      return waitForPathToExist(path, maxWaitTime - 100)
```
wrt Balazs's comment about, this should be a return statement